airflow-core/src/airflow/ui/src/layouts/Details/PanelButtons.tsx

```diff
@@ -72,10 +78,19 @@ const deps = ["all", "immediate", "tasks"];
 
 type Dependency = (typeof deps)[number];
 
-export const PanelButtons = ({ dagView, limit, panelGroupRef, setDagView, setLimit }: Props) => {
+export const PanelButtons = ({
+  dagView,
+  limit,
+  panelGroupRef,
+  setDagView,
+  setLimit,
+  setShowGantt,
+  showGantt,
+}: Props) => {
   const { t: translate } = useTranslation(["components", "dag"]);
-  const { dagId = "" } = useParams();
+  const { dagId = "", runId, taskId } = useParams();
   const { fitView } = useReactFlow();
+  const shouldShowToggleButtons = Boolean(runId ?? taskId);
```

Comment:
Shouldn't this just be `Boolean(runId)`?

In the future we can make a custom /gantt endpoint to generate a box-whisker plot to show trends across runs.